test/profiler/test_cpp_thread.cpp

```diff
@@ -106,4 +106,4 @@ PYBIND11_MODULE(profiler_test_cpp_thread_lib, m) {
       "start_threads",
       &start_threads,
       py::call_guard<py::gil_scoped_release>());
-};
+};
```

Comment:
nit: is this change expected?

Reply:
Thank you for review @aaronenyeshi 
This change is not expected. We restore the change here.
Thank you.